docs/index.md

```diff
@@ -2332,9 +2332,12 @@ $ mocha "./spec/**/*.js"
 ```
 
 [You should _always_ quote your globs in npm scripts][article-globbing]. If you
-use double quotes, it's the shell on UNIX that will expand the glob. On the
-other hand, if you use single quotes, the [`node-glob`][npm-glob] module will
-handle its expansion.
+use double quotes or single quotes, the [`node-glob`][npm-glob] module will
+handle its expansion. Single quotes are not supported on Windows, so you should
+avoid them. Note that UNIX shells will perform
```

Comment:
Trimming down: avoiding them is implied by the fact that they're not supported. We can let folks make their own conclusion there. Less docs -> more likely to be read.

```suggestion
handle its expansion. Single quotes are not supported on Windows.
Note that UNIX shells will perform
```